src/pages/guides/reference/configuringproxy.md

```diff
@@ -0,0 +1,15 @@
+# Configuring Proxy 
+
+Runtime does not expose egress IPs due to security reasons. If customers need a way to securely communicate with their backend services, they can use a proxy in between their system and Runtime. 
+
+This can be done as following: 
+
+1. Setting up an Azure Front Door
```

Comment:
It's not very clear why a customer should setup an `AFD` and what exactly should be set up.

Reply:
For now just adding options on what can be done will add more details.